docker/README.md

```diff
@@ -43,10 +43,16 @@ docker exec -it presto presto --catalog tpch --schema sf1
 
 ## Configuration
 
-Configuration is expected to be mounted to either  to `/etc/presto` or
+Configuration is expected to be mounted to either to `/etc/presto` or
```

Comment:
it would be best if we provided one option and keep things simple. /etc/presto  is simple and this is what we should use & recommend.